src/Framework/TestCase.php

```diff
@@ -383,6 +388,10 @@ public function __construct($name = null, array $data = [], $dataName = '')
             $this->setName($name);
         }
 
+        if (!isset(self::$DEPENDENCY_TASK_RESULTS)) {
+            self::$DEPENDENCY_TASK_RESULTS = new TestResult;
```

Comment:
It's required to initialize only once. Because it's static, it's required to initialize only once. This is required due optimization reasons I mentioned above.